assets/js/components/read-more/index.js

```diff
@@ -12,18 +12,27 @@ class ReadMore extends Component {
 		super( ...arguments );
 
 		this.state = {
-			isExpanded: true,
-			isClamped: false,
-			content: '.',
+			/**
+			 * This is true when read more has been pressed and the full review is shown.
+			 */
+			isExpanded: false,
+			/**
+			 * True if we are clamping content. False if the review is short. Null during init.
+			 */
+			isClamped: null,
+			/**
+			 * Content is passed in via children.
+			 */
+			content: props.children,
+			/**
+			 * Max height for a review.
+			 */
+			maxHeight: 0,
 		};
 
-		this.lineHeight = 0;
-		this.start = 0;
-		this.middle = 0;
-		this.end = 0;
-		this.originalContent = 1 === React.Children.count( props.children ) ? props.children : props.children.join( '' );
-
-		this.element = createRef();
+		this.maxHeight = 0;
```

Comment:
This is never used, right? I think we are always using the value from `this.state.maxHeight`.